.github/workflows/test.yml

```diff
@@ -19,5 +19,15 @@ jobs:
       - uses: actions/checkout@v2
       - name: install
         run: sudo apt-get update && sudo apt-get install aptitude qemu-kvm zstd binutils-dev elfutils libcap-dev libelf-dev libdw-dev python3-docutils
+      - name: Get bpf-next source
+        if: ${{ github.repository != 'kernel-patches/bpf' }}
```

Comment:
what's the longer-term solution for this? once we add net/net-next tree, for example, will we keep extending this hard-coded if?

Reply:
This is a good POC. The long term solution would be to keep .gh/wfs/test.yml solely for purpose of testing of CI repo itself and migrate all projects to dedicated folders : https://github.com/tsipa/ci/tree/master/custom-workflows/bpf